client/state/sites/selectors.js

```diff
@@ -10,3 +10,23 @@
 export function getSite( state, siteId ) {
 	return state.sites.items[ siteId ] || null;
 }
+
+/**
+ * Returns the slug for a site, or null if the site is unknown.
+ *
+ * @param  {Object} state  Global state tree
+ * @param  {Number} siteId Site ID
+ * @return {String}        Site slug
+ */
+export function getSiteSlug( state, siteId ) {
+	const site = getSite( state, siteId );
+	if ( ! site ) {
+		return null;
+	}
+
+	if ( site.options && site.options.is_redirect ) {
+		return site.options.unmapped_url.replace( /^https?:\/\//, '' );
+	}
+
+	return site.URL.replace( /^https?:\/\//, '' ).replace( /\//g, '::' );
+}
```

Comment:
>This differs from the current implementation in that `hasConflict` is not included:  https://github.com/Automattic/wp-calypso/search?q=hasConflict

Good eyes. I must have had my blinders on when I had overlooked that part of the condition ( :see_no_evil: ).

@enejb : There's quite a bit of background here. Do you think this is logic that needs to be preserved moving forward in the sites state? Apologies to shine light on this again, but I'm finding it difficult to find the conclusion of the previous discussion.